test/ui-test/src/screens/StatusWelcomeScreen.py

```diff
@@ -222,24 +222,29 @@ def profile_settings_image_is_updated(self):
         # first time clicking on settings button closes the my profile modal
```

Comment:
Unrelated to your changes but it seems this method should be in `StatusMainScreen` instead. I would appreciate if you update it accordingly! :)

Reply:
Moved all the profile verifications to StatusMainScreen